tests/filesystem/clean_path_test.php

```diff
@@ -1,54 +0,0 @@
-<?php
```

Comment:
https://github.com/rubencm/phpbb/blob/ticket/15253/tests/filesystem/helper_clean_path_test.php

Or should i restart the originals too?